apps/condo/domains/miniapp/components/AppFrameWrapper.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect } from 'react'
+import { css, Global } from '@emotion/react'
+import bridge from '@open-condo/bridge'
+
+const BODY_RESIZE_STYLES = css`
+  body {
+    height: auto;
+  }
+`
+
+export const AppFrameWrapper: React.FC = ({ children }) => {
+    useEffect(() => {
+        const observer = new ResizeObserver((entries) => {
+            if (entries && entries.length) {
+                bridge.send('CondoWebAppResizeWindow', { height: entries[0].target.clientHeight })
```

Comment:
`height` only? What about width?

Reply:
Width is always 100%.
Maybe you can specify small width by yourself, but in most cases it's redundant